{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/222/comments/37417501.json"}, "html": {"href": "#!/osrf/srcsim/issues/222#comment-37417501"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/222.json"}}, "type": "issue", "id": 222, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Local docker network error"}, "content": {"raw": "I deleted the --net and --ip to get by that error. Not sure what the overall impact is. May mean you can't SSH in directly or setup a VPN.", "markup": "markdown", "html": "<p>I deleted the --net and --ip to get by that error. Not sure what the overall impact is. May mean you can't SSH in directly or setup a VPN.</p>", "type": "rendered"}, "created_on": "2017-06-07T00:59:27.846157+00:00", "user": {"display_name": "Rud Merriam", "uuid": "{79efa10c-b8b6-4b55-9d05-7d0057e235cb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D"}, "html": {"href": "https://bitbucket.org/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc/5c7bde69-befb-417b-9aae-9db0c9a59777/128"}}, "nickname": "rmerriam", "type": "user", "account_id": "557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc"}, "updated_on": null, "type": "issue_comment", "id": 37417501}